eva/udfs/ndarray/annotate.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import cv2
+import numpy as np
+import pandas as pd
+
+from eva.udfs.abstract.abstract_udf import AbstractUDF
+
+color = (207, 248, 64)
+thickness = 4
+
+
+class Annotate(AbstractUDF):
+    def setup(self):
+        pass
+
+    @property
+    def name(self):
+        return "Annotate"
+
+    def forward(self, df: pd.DataFrame) -> pd.DataFrame:
+        """
+        Modify the frame to annotate the bbox on it.
+
+         Returns:
+             ret (pd.DataFrame): The modified frame.
+        """
+
+        def annotate(row: pd.Series) -> np.ndarray:
+            row = row.to_list()
+            frame = row[0]
+
+            bboxes = row[2]
```

Comment:
How do we pass bounding box information in a OVERWRITE query?

Reply:
If `OVERWRITE TABLE with Annotate(data, Yolov5(data));` works now, let's have a test case for that.